### DISCUSSION TITLE
Show responsive views within the editor

### DISCUSSION THREAD

**Discussion by zehawki** [react-page/react-page#960] [Ideas]
How about the ability to view 3 versions of the same page while its being edited, perhaps under "preview results" - one for wider screen, one for tablet and one for mobile. Right now all one sees is the wider screen view, and then perhaps turn on dev tools to see the other versions.

A much better / user friendly way would be to be able to switch between these 3 views via the side toolbar, without having to muck around with dev tools (which of course the layman knows nothing of).

Being able to see responsive views of the page while being created / edited is tremendously useful.

**Comment by macrozone** [comment]
@zehawki media queries only work properly if you mount it inside an iframe. this is possible with something like https://github.com/ryanseddon/react-frame-component

i did this once in a project, but i could not make i fully work.

its possible to do this in userspace, without any work from us. i convert to discussion
